src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.fruit.Operation;
+import core.basesyntax.fruit.Transaction;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.service.TransactionService;
+import java.util.List;
+import java.util.Map;
+
+public class TransactionServiceImpl implements TransactionService {
+    private final Map<Operation, OperationHandler> operationHandlerMap;
```

Comment:
You can use the Strategy instead of Map
```
public class OperationStrategyImpl implements OperationStrategy {
    private Map<Operation, OperationHandler> operationHandlerMap;

    public OperationStrategyImpl(Map<Operation,
            OperationHandler> operationHandlerMap) {
        this.operationHandlerMap = operationHandlerMap;
    }

    @Override
    public OperationHandler get(Operation operation) {
        return operationHandlerMap.get(operation);
    }
}
```